test/gnosis-safe.test.ts

```diff
@@ -144,4 +144,37 @@ describe('Gnosis Safe test', async () => {
       expect(sameTxs.length).to.be.equal(0);
     });
   });
+
+  it('Should return account details if valid address', async () => {
+    const accountDetails = await sdkInstance.safeGlobal.getAccountDetails(
+      SupportedChainId.MAINNET.toString(),
+      '0x2A83Aaf231644Fa328aE25394b0bEB17eBd12150'
+    );
+    expect(accountDetails).to.be.not.undefined;
+    expect(accountDetails?.network).to.be.equal(
```

Comment:
nit: to avoid having the `?` everywhere after `accountDetails` you just need to tell the compiler that it can't be undefined. The above expect doesn't actually do that for the compiler, so just do a `if(!accountDetails) throw Error("something");

Reply:
This while good in production somewhat clutters the test code with unnecessary ifs that imo hurt readability. I think ?. operator is fine here

Reply:
In the first expect(), exactly confirmed `accountDetails` is not `undefined`, but typescript compiler can't identity it.
In this case, do we need to have if condition before using `?`?

Reply:
@deep-quality-dev Yes we do, like what I have in my first comment. @RyanCross mentioned he would rather `?` than the `if` conditions, I have the opposite opinion, so it's your call.